docs/source/class_basics.rst

```diff
@@ -77,6 +77,53 @@ to it explicitly using ``self``:
            a = self
            a.x = 1      # Error: 'x' not defined
 
+Class and instance variable annotations
+***************************************
+
+Mypy supports annotations for class and instance 
+variables in class bodies and methods. Use ``ClassVar`` to
+indicate to the static type checker that this variable 
+should not be set on instances. 
+
+.. code-block:: python
+
+  class A:
+      y: ClassVar[Dict[str, int]] = {}  # class variable
+      z: int = 10                       # instance variable
+
+
+``ClassVar`` accepts only types and cannot be further subscribed.
+
+``ClassVar`` is not a class itself, and should not be used with 
+isinstance() or issubclass(). 
+
+``ClassVar`` does not change Python runtime behavior, but 
+it can be used by third-party type checkers. For example, a type checker 
```

Comment:
This section is kind of vague. It would be better to explain what mypy actually does rather than what it might do :-)